tests/eval_tests/utils.py

```diff
@@ -173,25 +261,33 @@ def subplots_comparison(df, column_name='tool_call_match',save_filename=True):
     if save_filename:
         save_plot(fig, f'Comparison of {column_name} by server type')
 
-def get_analysis_plots():
-        # Load the CSV file into a DataFrame
-    file_path = './results/metrics.csv'
-    df = pd.read_csv(file_path)
+def get_analysis_plots(subplot_comparison=True, per_tool_plot=False):
+    # Load the CSV files into DataFrames
+    mcp_metric_file_path = './results/metrics.csv'
+    client_metric_file_path = './results/client_tool_metrics.csv'
+    mcp_df = pd.read_csv(mcp_metric_file_path)
+    client_tool_df = pd.read_csv(client_metric_file_path)
```

Comment:
Instead of having these bool values, Why not make the `get_analysis_plots()` take a parameter "metrics_file" and then remove the hard coded file paths here?